src/midelware/globalErrorsMiddleware.js

```diff
@@ -0,0 +1,12 @@
+import { generalLogger as logger } from '../logger/logger'
+import { StatusCodes } from 'http-status-codes'
+import { INTERNAL_SERVER_ERROR } from '../error/errorMessages'
+
+const ERROR_MESSAGE = 'Error occurred.'
+
+function handleError (err, req, res) {
```

Comment:
I have run your project and get an error `res.status is not a function`. You have to add 4th parameter (next) to this function even if will not use it. By parameters count ExpressJS will recognize that this function is a global error handler

Reply:
Fixed